### PR TITLE
Fixed description link paths

### PR DESCRIPTION
**Pull request by imaustink**

### CODE REVIEW

**Comment by chasenlehara** [comment]
This doesn’t seem to be working for me:
not working

**Comment by imaustink** [comment]
@chasenlehara my bad! I clearly didn't test that good enough. Updated the PR 😄

File: static/search.js [hunk 1]
```diff
@@ -506,25 +506,32 @@ var Search = Control.extend({
...
 			},
-			addTargetToExternalURLs: function(html){
+			addTargetToExternalURLs: function(html, parentHref){
 				var $html = $('<div>').html(html);
 				$html.find('a').each(function(){
 					var $a = $(this);
 					var a = $a[0];
+					var isLocal = a.hostname === location.hostname;
+					var isRelative = a.getAttribute('href').indexOf(location.hostname) === -1;
 					if(a.hostname !== location.hostname || a.protocol !== location.protocol){
```
Comment (chasenlehara) [inline static/search.js]: `a.hostname !== location.hostname` could be `!isLocal`

File: static/search.js [hunk 2]
```diff
@@ -506,25 +506,32 @@ var Search = Control.extend({
...
 						$a.attr('target', '_blank');
 					}
+					if(isLocal && isRelative){
+						var prefix = parentHref.substr(0, parentHref.lastIndexOf('/'));
```
Comment (chasenlehara) [inline static/search.js]: Comments on lines like this would make the code easier to read.

File: static/search.js [hunk 3]
```diff
@@ -506,25 +506,32 @@ var Search = Control.extend({
...
-			docUrl: function(){
+			docUrl: function(url){
 				if(self.options.pathPrefix){
-					return self.options.pathPrefix + "/" + this.url;
+					return self.options.pathPrefix + "/" + url;
 				}
 
-				if(this.url.substr(-1) === "/"){
+				if(url.substr(-1) === "/"){
 					return this.url;
```
Comment (chasenlehara) [inline static/search.js]: Should this be just `url`?

File: static/search.js [hunk 4]
```diff
@@ -506,25 +506,32 @@ var Search = Control.extend({
...
 
-				return "/" + this.url;
+				return "/" + url;
 			},
-			addTargetToExternalURLs: function(html){
+			addTargetToExternalURLs: function(html, parentHref){
 				var $html = $('<div>').html(html);
 				$html.find('a').each(function(){
 					var $a = $(this);
 					var a = $a[0];
```
Comment (chasenlehara) [inline static/search.js]: Going through this dance to convert the object to be jQuery-wrapped and then to unwrap it seems odd.

File: static/search.js (hunk 3 above)
Comment (imaustink) [inline static/search.js]: Yes, I added a URL argument here: https://github.com/canjs/bit-docs-html-canjs/pull/357/files/1eec29018d58105f40c63d39f6a95f8bd71a591f#diff-2d66ff3204071453e9b0e8e7ad4403ebR509

**Pull request merged**